concent_api/core/tests/utils.py

```diff
@@ -163,6 +163,13 @@ def _sign_message(self, golem_message, client_private_key = None):
             self.REQUESTOR_PRIVATE_KEY if client_private_key is None else client_private_key,
         )
 
+    def _generate_ethereum_signature(self, task_to_compute, requestor_ethereum_private_key=None):
```

Comment:
I see that sometimes ethereum signature is created "manually" (by call to `task_to_compute.generate_ethsig(REQUESTOR_PRIV_ETH_KEY)`) and sometimes by the function above. Why not having it unified?

Reply:
In most cases it is automatically done in `_get_deserialized_task_to_compute`. But in few auth tests we change something in returned `TaskToCompute`, so signature has to be regenerated manually.